src/DeploymentUtils.ts

```diff
@@ -1,19 +1,47 @@
-import * as deployments from "../deployments/deployments.json";
+import * as deployments_ from "../deployments/deployments.json";
+interface DeploymentExport {
+  [chainId: string]: { [contractName: string]: { address: string; blockNumber: number } };
+}
+const deployments: DeploymentExport = deployments_ as any;
 
 // Returns the deployed address of any contract on any network.
 export function getDeployedAddress(contractName: string, networkId: number): string {
   try {
-    return (deployments as any)[networkId.toString()][contractName].address;
+    return deployments[networkId.toString()][contractName].address; // First try match directly on the name provided.
   } catch (_) {
+    try {
+      // If this does not match then try search on a Regex on SpokePool to let the caller exclude the chain name.
+      for (const _contractName of Object.keys(deployments[networkId.toString()]))
+        if (/.*_SpokePool/.test(_contractName)) return deployments[networkId.toString()][_contractName].address;
+    } catch (_) {}
     throw new Error(`Contract ${contractName} not found on ${networkId} in deployments.json`);
   }
 }
 
 // Returns the deployment block number of any contract on any network.
-export function getDeployedBlockNumber(contractName: string, networkId: number): string {
+export function getDeployedBlockNumber(contractName: string, networkId: number): number {
   try {
-    return (deployments as any)[networkId.toString()][contractName].blockNumber;
+    return deployments[networkId.toString()][contractName].blockNumber;
```

Comment:
this should have the same behavior as getDeployedAddress.  imo we might just want a way to look up the correct contract name, then you dont have to modify thsese 2 functions at all

Reply:
fixed. see above comments